core-web/apps/dotcms-binary-field-builder/project.json

```diff
@@ -41,6 +41,11 @@
                 ]
             },
             "configurations": {
+                "localhost": {
```

Comment:
I don't think this is a good name for a configuration, shouldn't be like `development`

Reply:
Deleted, used development instead.